src/main/webapp/script.js

```diff
@@ -0,0 +1,37 @@
+
+function getDatabases(){
+ 
+    fetch("/example").then(response => response.json()).then((list) => {
+      console.log(list);
+
+      let dropdown = document.getElementById('list-databases');
+      dropdown.length = 0;
```

Comment:
This seems odd. I am guessing you are doing this to clear the options set in the dropdown?

I would suggest `dropdown.options.remove(0)` as it is considerably easier to read.

Reply:
Yes that's what it does, I wasn't familiar with this other option you showed but I do see how it's easier to read! Thanks for the tip just changed it!